plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/bedrock-runtime.ts

```diff
@@ -102,6 +102,13 @@ export class BedrockRuntimeServiceExtension implements ServiceExtension {
         return this.requestPreSpanHookConverse(request, config, diag, true);
       case 'InvokeModel':
         return this.requestPreSpanHookInvokeModel(request, config, diag);
+      case 'InvokeModelWithResponseStream':
+        return this.requestPreSpanHookInvokeModelWithResponseStream(
```

Comment:
Is there a reason to not re-use `requestPreSpanHookInvokeModel`: add a 4th `isStream` argument and pass in `false` for 'InvokeModel', `true` for 'InvokeModelWithResponseStream', and then make the minor update to the implementation?  This is how it was done for 'Converse' and 'ConverseStream'.

It looks to me like the `requestPreSpanHookInvokeModel` and `requestPreSpanHookInvokeModelWithResponseStream` functions are almost identical ... except that the latter doesn't have blocks for 'meta.llama', 'cohere.*', and 'mistral'.